source/SnapDump-Server-Tests/SDHTTPTests.class.st

```diff
@@ -9,28 +9,24 @@ Class {
 }
 
 { #category : #accessing }
-SDHTTPTests >> apiHandler [		
-	^ ZnPrefixMappingDelegate new 
-		map: 'api' 
-		to: (SDServerDelegate new
-			store: self filesystemStore;
-			uriSpace: (OpenAPIUriSpace new
-				rootClass: SDOpenAPICall))
+SDHTTPTests >> apiUri [
+	^ ('http://', self selfIP, ':', self randomPort asString, '/api') asZnUrl 
 ]
 
 { #category : #accessing }
-SDHTTPTests >> apiUri [
-	^ ('http://', self selfIP, ':', self randomPort asString, '/api') asZnUrl 
+SDHTTPTests >> apiVersionedUri [
+
+	^ self apiUri / SDSnapshot apiVersion
 ]
 
 { #category : #tests }
 SDHTTPTests >> dummySnapshotUrl [
-	^ (self apiUri /#projects / self projectNameForTests / #versions / self versionStringForTests / #exceptions / self dummyExceptionId / #snapshots / self dummySnapshotId)
+	^ (self apiVersionedUri /#projects / self projectNameForTests / #versions / self versionStringForTests / #exceptions / self dummyExceptionId / #snapshots / self dummySnapshotId)
```

Comment:
Just a side note. The API uri is the one that contains the v2. There is no other version like versioned. But I'm also not against it

Reply:
Good point, less confusion is better :P

Reply:
Ok I could check for a better naming then.
The main point to understand with this change is:

The method #apiVersion was used to:
1) set url to the http store to be used by test
2) build all kind of "standalone" urls used by tests

The first case does not expect the "api version" to be part of the url. It will be automatically added each time the store tries to reach snapdump server
The second case does expect the "api version".  These urls will be directly used with custom ZnClients (not going through a http store)

Reply:
Ok, got it